<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send File</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>280</width>
      <height>293</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 40px; /* Larger font size */
    font-weight: bold; /* Make the text bold */
    color: #FF4500; /* Red text color (example: DarkOrange) */
    background: transparent; /* No background */
    text-align: center; /* Center the text horizontally */
}
</string>
       </property>
       <property name="text">
        <string>Send File</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 30px; /* Larger font size */
    font-weight: bold; /* Make the text bold */
    color: #FF4500; /* Red text color (example: DarkOrange) */
    background: transparent; /* No background */
    text-align: center; /* Center the text horizontally */
}
</string>
         </property>
         <property name="text">
          <string>IP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="ip_text_edit">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>37</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QTextEdit {
    background-color: #333; /* Dark gray background color for QTextEdit */
    color: #FFF; /* White text color */
    border: 2px solid #FF4500; /* Red border around the QTextEdit */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Padding for text content */
}

QTextEdit:hover {
    border: 2px solid #FF6600; /* Red border on hover */
}

QTextEdit:focus {
    border: 2px solid #FF3300; /* Red border when QTextEdit is in focus */
}
</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 30px; /* Larger font size */
    font-weight: bold; /* Make the text bold */
    color: #FF4500; /* Red text color (example: DarkOrange) */
    background: transparent; /* No background */
    text-align: center; /* Center the text horizontally */
}
</string>
         </property>
         <property name="text">
          <string>PORT:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="port_text_edit">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>38</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QTextEdit {
    background-color: #333; /* Dark gray background color for QTextEdit */
    color: #FFF; /* White text color */
    border: 2px solid #FF4500; /* Red border around the QTextEdit */
    border-radius: 5px; /* Rounded corners */
    padding: 3px; /* Padding for text content */
}

QTextEdit:hover {
    border: 2px solid #FF6600; /* Red border on hover */
}

QTextEdit:focus {
    border: 2px solid #FF3300; /* Red border when QTextEdit is in focus */
}
</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="set_conn_button">
         <property name="maximumSize">
          <size>
           <width>420</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QMainWindow QPushButton {
    background-color: #CC3300; /* Darker red background color */
    color: #FFF; /* White text color */
    border: none; /* Remove the button border */
    border-radius: 20px; /* Make the button round by increasing border-radius */
    padding: 10px; /* Adjust padding for button content */
}

QMainWindow QPushButton:hover {
    background-color: #FF4500; /* Lighter red background color on hover */
}

QMainWindow QPushButton:pressed {
    background-color: #AA2200; /* Even darker red background color when pressed */
}</string>
         </property>
         <property name="text">
          <string>Set Connection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="conn_status_box">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">

QCheckBox::indicator::unchecked {
    background-color:red ;
}

</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>271</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="dir_label">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 15px; /* Larger font size */
    color: #FF4500; /* Red text color (example: DarkOrange) */
    background: transparent; /* No background */
    text-align: center; /* Center the text horizontally */
}
</string>
       </property>
       <property name="text">
        <string>C:/</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="choose_dir_button">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QMainWindow QPushButton {
    background-color: #CC3300; /* Darker red background color */
    color: #FFF; /* White text color */
    border: none; /* Remove the button border */
    border-radius: 20px; /* Make the button round by increasing border-radius */
    padding: 10px; /* Adjust padding for button content */
}

QMainWindow QPushButton:hover {
    background-color: #FF4500; /* Lighter red background color on hover */
}

QMainWindow QPushButton:pressed {
    background-color: #AA2200; /* Even darker red background color when pressed */
}</string>
       </property>
       <property name="text">
        <string>Choose File to Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="send_file_button">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QMainWindow QPushButton {
    background-color: #CC3300; /* Darker red background color */
    color: #FFF; /* White text color */
    border: none; /* Remove the button border */
    border-radius: 20px; /* Make the button round by increasing border-radius */
    padding: 10px; /* Adjust padding for button content */
}

QMainWindow QPushButton:hover {
    background-color: #FF4500; /* Lighter red background color on hover */
}

QMainWindow QPushButton:pressed {
    background-color: #AA2200; /* Even darker red background color when pressed */
}</string>
       </property>
       <property name="text">
        <string>Send File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="sending_progress_bar">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
    border: 2px solid #FF4500; /* Add a red border */
    border-radius: 10px; /* Make the progress bar rounded */
    text-align: center; /* Center the text inside the progress bar */
    font-size: 14px; /* Set the font size */
    color: #FFF; /* Set the text color to white */
}

QProgressBar::chunk {
    background-color: #FF4500; /* Set the progress bar chunk color to red */
}</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
